src/views/template/history/customer.html

```diff
@@ -0,0 +1,13 @@
+<li class="list-group-item">
+    <div class="panel panel-default">
+        <div class="panel-heading">{{ name }}</div>
```

Comment:
English names to Dutch equivalents. Or two sets of names. I'm not too sure, but something along those lines, yes.